docs/local-k8s-setup.md

```diff
@@ -3,6 +3,7 @@
 ### Prerequisites
 
 Have [Docker for Desktop](https://www.docker.com/products/docker-desktop) installed and running 
+Have [WSL Feature Installed](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and configured
```

Comment:
I don't think its actually needed to get local k8s up and running, but I'm willing to close my eye as I think WSL is a nice to have for any Win-only dev 😄

Reply:
Removed and updated documentation